gameboard.js

```diff
@@ -205,7 +205,7 @@ function hideControlMessage() {
  */
 function startGame(game) {
   game.initializeSceneManager();
-  playAudio(1);
+  // playAudio(1);
```

Comment:
Think audio works on bleeding now, its safe to undo this

Reply:
you're right, resolved